### PR TITLE
Apollo GraphQL, RPDC Mutation, formatting

### PR DESCRIPTION
**Pull request by andricDu**
Finishes RDPC Client Scaffold: https://github.com/icgc-argo/workflow-graph-node/issues/8

### Changes
- Removes WesClient
- RdpcClient supports starting a workflow run
- Introduced Apollo client with code generation for type safe and null safe access to RDPC API Gateway
- Ran Google for formatter on project

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,8 +14,12 @@
     <name>workflow-graph-node</name>
     <description>Workflow Graph Node</description>
 
-    <dependencies>
+    <properties>
+        <apollo.version>2.3.1</apollo.version>
+        <okio.version>2.8.0</okio.version>
+    </properties>
```
Comment (lepsalex) [inline pom.xml:20]: should roll these into the parent pom

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -8,8 +8,8 @@ rabbit:
 
 logging.level.root: INFO
 
-wes:
-  url: http://localhost:9090/runs
+rdpc:
+  url: https://api.rdpc-dev.cancercollaboratory.org/graphql
```
Comment (lepsalex) [inline src/main/resources/application.yml:12]: do we want rdpc dev as default or leave localhost to be overwritten by dev?

**Review by lepsalex — APPROVED** [review]
Approved with comments, up to you if you want to address these now or later.

File: src/main/resources/application.yml (hunk 2 above)
Comment (andricDu) [reply src/main/resources/application.yml:12]: Might as well leave it at localhost, will change.

File: pom.xml (hunk 1 above)
Comment (andricDu) [reply pom.xml:20]: Good idea.

**Pull request merged**